src/analyzer/model.ts

```diff
@@ -75,7 +75,7 @@ export interface HydratedGitBlobObject extends GitBlobObject {
   lastChangeEpoch?: number
   isBinary?: boolean
   unionedAuthors?: Record<AuthorshipType, Record<string, number>>
-  dominantAuthor?: Map<AuthorshipType, [string, number]>
+  dominantAuthor?: Record<AuthorshipType, [string, number]>
```

Comment:
As `Record` does have not order options we can reply on -[Discussion on StackOverflow]( https://stackoverflow.com/questions/70781123/how-to-prevent-typescript-recordstring-any-from-sorting-object-properties-to). 

Would we not need to convert it to Map or array at some point? Is this change needed to make lazy load repositories work?

Reply:
This was necessary when moving away from typedjson, which enabled using remix `defer` since we no longer revive the JSON and can only use JSON primitives. The ordering of this does not matter, since it was only used for switching between git blame and all history (which is currently not enabled), so I think this change is fine.

Reply:
I don't have rights to resolve the conversation but you answered my concern, so it should be resolved.